src/chain/chain_accumulator.rs

```diff
@@ -48,6 +93,13 @@ impl ChainAccumulator {
         proof: Proof,
         signature: Option<SectionInfoSigPayload>,
     ) -> Result<(), InsertError> {
+        // The entry must be pre-existed. Otherwise just don't track.
+        let voters: Vec<_> = iter::once(proof.pub_id).collect();
+        let _ = self
+            .vote_statuses
+            .iter_mut()
+            .any(|vote_status| vote_status.add_votes(&event, &voters));
```

Comment:
Why is this not only for completed votes?

Reply:
We need to update the tracker so a lagging voter won't be counted as non-responsive.

Reply:
Sorry, could you explain how this is related?

Reply:
On consensus, there is super-majority voters included in the `proofs`, from which we can collect the voters info.
For other `lagging` voters, we have to update tracker via this add_proof function.
Otherwise, the tracker will think those voters are `non-responsive`, and we are actually punishing the laggers.

Reply:
Can you explain in what situation will `vote_status.add_votes` actually do something for an event that is not in completed_events?